e2e/tests/components/number-picker/readonly/number-picker-read-only.e2e-spec.ts

```diff
@@ -0,0 +1,93 @@
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 5 }');
+        });
+
+        it('should prevent changes to the value by hitting the decrement arrow', async () => {
+            await browser.actions().mouseMove(page.decrementArrow).click().perform();
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 5 }');
+        });
+
+        it('should prevent changes to the value by hitting the up arrow on the keyboard', async () => {
+            await browser.actions().mouseMove(page.numberPickerReadOnly).click().perform();
+            await browser.actions().sendKeys(Key.ARROW_UP).perform();
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 5 }');
+        });
+
+        it('should prevent changes to the value by hitting the down arrow on the keyboard', async () => {
+            await browser.actions().mouseMove(page.numberPickerReadOnly).click().perform();
+            await browser.actions().sendKeys(Key.ARROW_DOWN).perform();
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 5 }');
+        });
+    });
+
+    describe('false', () => {
+
+        beforeAll(async () => {
+            await page.clickOnCheckbox(page.readonlyCheckbox);
+        });
+
+        beforeEach(async () => {
+            await page.setNumberPickerValue(page.numberPickerReadOnly, '5');
+        });
+
+        it('should allow changes to the value via the input', async () => {
+            await page.setNumberPickerValue(page.numberPickerReadOnly, '7');
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 7 }');
+        });
+
+        it('should allow changes to the value by hitting the increment arrow', async () => {
+            await browser.actions().mouseMove(page.incrementArrow).click().perform();
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 6 }');
+        });
+
+        it('should allow changes to the value by hitting the decrement arrow', async () => {
+            await browser.actions().mouseMove(page.decrementArrow).click().perform();
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 4 }');
+        });
+
+        it('should allow changes to the value by hitting the up arrow on the keyboard', async () => {
+            await browser.actions().mouseMove(page.numberPickerReadOnly).click().perform();
+            await browser.actions().sendKeys(Key.ARROW_UP).perform();
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 6 }');
+        });
+
+        it('should allow changes to the value by hitting the down arrow on the keyboard', async () => {
+            await browser.actions().mouseMove(page.numberPickerReadOnly).click().perform();
+            await browser.actions().sendKeys(Key.ARROW_DOWN).perform();
+
+            expect(await page.emittedValue.getText()).toBe('{ "readonly": 4 }');
+        });
+    });
```

Comment:
I would hope that these are covered by the existing e2e tests for number picker, so it should be safe to omit them. Bear in mind that the e2e tests are extremely slow to run, so we only want to add tests which definitely add value.